boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp_0_2_1.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54l15pdk@0.2.1/nrf54l15/cpuapp
+name: nRF54l15-PDK-nRF54l15-Application
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 256
+flash: 1536
```

Comment:
Are those values correct? I think those values should be equal to the ones specified in `nrf54l15pdk_nrf54l15_cpuapp.yaml`, as that is how much RAM and NVM is assigned to APP core.

Reply:
Thank you, I have updated the numbers for all the boards